<?xml version="1.1"?>
<launch>
	<node name="errorManager" pkg="dependency_pkg" type="errorManager" output="screen" />

	<node name="nodeKiller" pkg="dependency_pkg" type="nodeReinitialiser" output="screen" />

	<node name="stateRecoverer" pkg="dependency_pkg" type="stateRecoverer" output="screen" />

	<node name="watchdogInterface" pkg="dependency_pkg" type="watchdogInterface" output="screen" />

</launch>
